app/src/main/java/org/xbmc/kore/jsonrpc/type/PlayerType.java

```diff
@@ -283,6 +283,19 @@ public PositionTime(int hours, int minutes, int seconds, int milliseconds) {
             this.milliseconds = milliseconds;
         }
 
+        /**
+         * Converts second duration into a hour, minutes, seconds duration
+         * @param seconds
+         */
+        public PositionTime(int seconds) {
+            this.hours = seconds / 3600;
+            int remainder = seconds - (this.hours * 3600);
```

Comment:
Using modulo to get the remaining seconds would require an additional division, which is more expensive than multiplication and subtraction.